lib/msal-common/src/utils/Constants.ts

```diff
@@ -64,6 +64,13 @@ export const Constants = {
     INVALID_INSTANCE: "invalid_instance",
 };
 
+export enum ERROR_CODE_BOUNDS {
```

Comment:
Please define this as an object literal instead (look at HeaderNames or PersistentCacheKeys a few lines below). We stopped using the native enums due to size concerns

Reply:
interesting!